src/tree/localProject/ConfigGroupTreeItem.ts

```diff
@@ -28,12 +30,12 @@ export class ConfigGroupTreeItem extends AzExtParentTreeItem {
     }
 
     public async loadMoreChildrenImpl(): Promise<AzExtTreeItem[]> {
-        const buildSettings: string[] = ['app_location', 'api_location', 'app_artifact_location'];
+        const buildConfigs: BuildConfig[] = ['app_location', 'api_location', 'output_location'];
```

Comment:
We actually do need to support both output and app_artifact since older yamls use `app_artifact_location`